docs/content/contributing/coding-style.md

```diff
@@ -115,7 +115,7 @@ A positive example, which is taken from `setup-stack.sh`, would be
 ```bash
 function _setup_postfix_aliases
 {
-  _notify 'task' 'Setting up Postfix Aliases'
+  _log 'task' 'Setting up Postfix Aliases'
```

Comment:
```suggestion
  _log 'debug' 'Setting up Postfix aliases'
```

Reply:
👍🏼

Reply:
Done in f097b1d08e40259d9d558b79a9875518292a9466